pack-contracts/pack-contract-grpc/src/main/proto/GrpcTxEvent.proto

```diff
@@ -33,6 +33,7 @@ message GrpcServiceConfig {
 }
 
 message GrpcAck {
+  bool valid = 1;
```

Comment:
valid doesn't seem to be a proper name, because the transaction is indeed valid. probably just `aborted`?